src/main/scala/org/ergoplatform/nodeView/wallet/persistence/Balance.scala

```diff
@@ -1,13 +1,14 @@
 package org.ergoplatform.nodeView.wallet.persistence
 
-import org.ergoplatform.nodeView.wallet.IdUtils._
+import org.ergoplatform.ErgoBox.BoxId
 import org.ergoplatform.wallet.boxes.TrackedBox
+import scorex.util.{ModifierId, bytesToId}
 
-case class Balance(id: EncodedBoxId,
+case class Balance(id: BoxId,
```

Comment:
@jellymlg, after EncodedBoxId is changed to BoxId, the equality of case class became broken, as BoxId is Array. Need to override `equals` and `hashCode`. Why EncodedBoxId was replaced?